<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShcDlg</class>
 <widget class="QDialog" name="ShcDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit { 
    border: 1px solid rgb(41, 57, 85);
    background: white;
    selection-background-color: darkgray;
    font-size: 16px ;
}

QLineEdit:hover {
	border: 2px solid blue;
}

QLineEdit[echoMode=&quot;2&quot;] {
      lineedit-password-character: 42;
}

QPushButton{
	border-radius:6px;
	background-color: #1390C5;
	min-width:80px;
}

QPushButton:pressed {
	background-color: #105D7E;
}

QPushButton:hover {
	background-color: #2BA0F3;
}

color: rgb(211, 211, 211);

</string>
  </property>
  <widget class="QLineEdit" name="paymentEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>72</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 11pt &quot;微软雅黑 Light&quot;;</string>
   </property>
   <property name="text">
    <string>数量:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>46</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QPushButton" name="PayBtn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>93</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 11pt &quot;微软雅黑 Light&quot;;</string>
   </property>
   <property name="text">
    <string>书名:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>93</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>72</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 11pt &quot;微软雅黑 Light&quot;;</string>
   </property>
   <property name="text">
    <string>总金额:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>ShcDlg</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
